tools/toolchain/scripts/install_cosma.sh

```diff
@@ -0,0 +1,176 @@
+#!/bin/bash -e
+[ "${BASH_SOURCE[0]}" ] && SCRIPT_NAME="${BASH_SOURCE[0]}" || SCRIPT_NAME=$0
+SCRIPT_DIR="$(cd "$(dirname "$SCRIPT_NAME")" && pwd -P)"
+
+cosma_ver="2.0.6"
+cosma_sha256="25c73035628f5652a4df99881993c51af5806c9a7ecb27a9b0b9bea78c4f45f4"
+source "${SCRIPT_DIR}"/common_vars.sh
+source "${SCRIPT_DIR}"/tool_kit.sh
+source "${SCRIPT_DIR}"/signal_trap.sh
+source "${INSTALLDIR}"/toolchain.conf
+source "${INSTALLDIR}"/toolchain.env
+
+[ -f "${BUILDDIR}/setup_cosma" ] && rm "${BUILDDIR}/setup_cosma"
+
+! [ -d "${BUILDDIR}" ] && mkdir -p "${BUILDDIR}"
+cd "${BUILDDIR}"
+
+case "$with_cosma" in
+    __INSTALL__)
+	require_env OPENBLASROOT
+	require_env SCALAPACKROOT
+        export
```

Comment:
Left over debug line.

Reply:
thanks. I will correct it next patch. Waiting for the results of the regtests.